tests/Unit/Http/Controllers/ClientControllerTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Tests\Unit\Http\Controllers;
+
+use App\Http\Controllers\ClientController;
+use App\Models\Category;
+use App\Models\Post;
+use App\Models\Like;
+use App\Repositories\Category\CategoryRepository;
+use App\Repositories\Category\CategoryRepositoryInterface;
+use App\Repositories\Like\LikeRepository;
+use App\Repositories\Like\LikeRepositoryInterface;
+use App\Repositories\Post\PostRepository;
+use App\Repositories\Post\PostRepositoryInterface;
+use Exception;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use InvalidArgumentException;
+use Tests\TestCase;
+use Mockery as m;
+use RuntimeException;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class ClientControllerTest extends TestCase
+{
+    /**
+     * A basic unit test example.
+     *
+     * @return void
+     */
+    public function setUp() : void
+    {
+        parent::setUp();
+
+        $this->categoryRepo = m::mock(CategoryRepositoryInterface::class)->makePartial();
+        $this->postRepo = m::mock(PostRepositoryInterface::class)->makePartial();
+        $this->likeRepo = m::mock(LikeRepositoryInterface::class)->makePartial();
+        $this->clientController = new ClientController($this->categoryRepo, $this->postRepo, $this->likeRepo);
+    }
+
+    public function tearDown() : void
+    {
+        unset($this->categoryRepo);
+        unset($this->postRepo);
+        unset($this->likeRepo);
```

Comment:
```suggestion
        m::close();
```
Để ngắn gọn có thể unset các instance của **Mockery** bằng hàm **close()** nhé ;)

Reply:
ok